tests/components/component-playground/selected-fixture-and-editor/events/handlers.js

```diff
@@ -59,5 +60,15 @@ describe(`ComponentPlayground (${FIXTURE}) Events Handlers`, function() {
       expect(stateSet.fixtureUserInput)
             .to.equal(JSON.stringify(fixture.state.fixtureContents, null, 2));
     });
+
+    it('should set', function() {
```

Comment:
You should detail this a bit more. It isn't that obvious what this should test from the fixture name.